### PR TITLE
Make podcasts more mobile friendly and use badges

### PR DESCRIPTION
**Pull request by thibaultamartin**

### CODE REVIEW

**Comment by thibaultamartin** [comment]
Screenshot 2022-11-02 at 13 12 11

The badges are looking quite sad atm. It's a pity they're not the same size. They certainly need some margin, and a consistent height. Not sure if we can do much more?

**Comment by thibaultamartin** [comment]
https://user-images.githubusercontent.com/1688309/199664910-6bdb3a7f-8247-463f-be66-ddf55f69bcce.mov

I think I managed to get something decent

File: templates/podcasts.html [hunk 1]
```diff
@@ -2,14 +2,19 @@
...
-    <div class="podcasts-grid">
-        <div>
+    <div class="podcasts-content">
+        <div class="podcasts-column">
             {{ matrix_live | markdown | safe }}
+            <div class="badges">
+                <a href="https://podcasts.apple.com/gb/podcast/matrix-live/id1498631706"><img src="/assets/apple_podcasts_badge.svg" alt="Listen on Apple Podcasts" /></a>
+                <a href="https://open.spotify.com/show/1SFrZPISzYs0elxlZUAvZ5?si=Xyns2Pl0RBuGBjuzH4yxsQ"><img src="/assets/spotify_podcasts_badge.svg" alt="Listen on Spotify" /></a>
+                <a href="https://pca.st/uv1qytez"><img src="/assets/pocketcasts_badge.svg" alt="Listen on Pocket Casts" /></a>
+            </div>
```
Comment (jplatte) [inline templates/podcasts.html:15]: Why is this part of the template rather than part of the content?

File: sass/_otwsu.scss [hunk 2]
```diff
@@ -1,7 +1,26 @@
-.podcasts-grid {
-    display: grid;
-    grid-template-columns: 2fr 2fr;
-    gap: 20px;
+.podcasts-content {
+    display: flex;
+    @media (max-width: 767px) {
+        flex-direction: column;
+    }
+    max-width: 1000px;
```
Comment (jplatte) [inline sass/_otwsu.scss:6]: Was 1200px too wide? I think a uniform wide page width would be nice.

File: sass/_otwsu.scss [hunk 3]
```diff
@@ -1,7 +1,26 @@
...
+    margin: 0 auto 2em auto;
+    justify-content: space-evenly;
```
Comment (jplatte) [inline sass/_otwsu.scss:8]: Does this actually do anything?

File: sass/_otwsu.scss [hunk 4]
```diff
@@ -1,7 +1,26 @@
...
-    display: grid;
-    grid-template-columns: 2fr 2fr;
-    gap: 20px;
+.podcasts-content {
+    display: flex;
+    @media (max-width: 767px) {
+        flex-direction: column;
+    }
+    max-width: 1000px;
+    margin: 0 auto 2em auto;
```
Comment (jplatte) [inline sass/_otwsu.scss:7]:
```suggestion
    margin: 0 auto 2em;
```
Has the same meaning.

File: templates/podcasts.html (hunk 1 above)
Comment (thibaultamartin) [reply templates/podcasts.html:15]: Hm, do you suggest to add markdown to the matrix_live partial or to put the podcasting platform links into a toml file (with path to image, link, and alt text), or to create an additional partial for badges?

Comment (jplatte) [reply templates/podcasts.html:15]: I was suggesting to add markdown to the partial.

Comment (thibaultamartin) [reply templates/podcasts.html:15]: I'm not sure how we could put the platform links in a div, if it was part of the partial content? There's a bit of CSS for this div in particular, to make spacing and wrapping look correct

Comment (jplatte) [reply templates/podcasts.html:15]: You can write HTML in markdown. You *might* have to put in extra newlines for the markdown img / link syntax to work, as in

```md
<div class="badges">

[![Listen on Apple Podcasts](/assets/apple_podcasts_badge.svg)](https://podcasts.apple.com/gb/podcast/matrix-live/id1498631706)
<!-- other two links ->

</div>
```

or define a shortcode like

```md
{% div(class="badges") %}
[![Listen on Apple Podcasts](/assets/apple_podcasts_badge.svg)](https://podcasts.apple.com/gb/podcast/matrix-live/id1498631706)
<!-- other two links ->
{% end %}

Comment (thibaultamartin) [reply templates/podcasts.html]: I had to use shortcodes to generate the `<a></a>`, zola would choke on image in link markdown

**Review by jplatte — APPROVED** [review]

**Review by Jhp1983 — CHANGES_REQUESTED** [review]
Eat my divk

**Pull request merged**